src/Passport/Bridge/RefreshTokenRepository.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace DesignMyNight\Mongodb\Passport\Bridge;
+
+use Laravel\Passport\Bridge\RefreshTokenRepository as BaseRefreshTokenRepository;
+use Laravel\Passport\Events\RefreshTokenCreated;
+use League\OAuth2\Server\Entities\RefreshTokenEntityInterface;
+use MongoDB\BSON\UTCDateTime;
+
+/**
+ * Class RefreshTokenRepository
+ * @package App\Passport\Bridge
+ */
+class RefreshTokenRepository extends BaseRefreshTokenRepository
+{
+    /**
+     * @inheritDoc
+     */
+    public function persistNewRefreshToken(RefreshTokenEntityInterface $refreshTokenEntity)
+    {
+        $this->refreshTokenRepository->create([
+            'id' => $id = $refreshTokenEntity->getIdentifier(),
+            'access_token_id' => $accessTokenId = $refreshTokenEntity->getAccessToken()->getIdentifier(),
+            'revoked' => false,
+            'expires_at' => new UTCDateTime($refreshTokenEntity->getExpiryDateTime()->getTimestamp() * 1000),
```

Comment:
Wouldn't this line also work with just `$refreshTokenEntity->getExpiryDateTime()`

Reply:
Unfortunately not, the `expires_at` gets set a string value if I don't pass a `UTCDateTime` instance.

Reply:
That would not be needed, if you would just alias and overwrite the RefreshToken model, I guess.

Reply:
If you mean making `getExpiryDateTime()` return a `UTCDateTime`, I don't think it would be a good idea to overwrite the return type in case a `DateTime` is expected elsewhere.